src/current/v23.1/role-based-audit-logging.md

```diff
@@ -0,0 +1,147 @@
+
+To store the audit log files in a specific directory, [configure the `SENSITIVE_ACCESS` channel]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files) with a custom `dir` path.
+
+{{site.data.alerts.callout_success}}
+If your deployment requires particular lifecycle and access policies for audit log files, point `SENSITIVE_ACCESS` to a directory that has permissions set so that only CockroachDB can create/delete files.
+{{site.data.alerts.end}}
+
+## Examples
+
+### Exclude one role from logging
+
+With the audit settings in this example,
+
+- Users with the username or role `service_account_role` will not emit audit logs for any statements they issue.
+- All remaining users will emit audit logs for any statements they issue. `ALL` is used twice. The first `ALL` refers to all `USER/ROLE`s. The second `ALL` refers to logging all actions/statements.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.log.user_audit = '
+    service_account_role NONE
+    ALL ALL
+';
+~~~
+
+### Include only one role for logging
+
+With the audit settings in this example,
+
+- Users with the username or role `test_role` will emit audit logs for all statements they issue.
+- Users with the username or role `another_role` will not emit audit logs for any statements they issue. In principle, you can achieve this by simply omitting `another_role` from the configuration entirely. It is included here as a basis of comparison to the next example - [User with multiple roles](#user-with-multiple-roles).
+- All remaining users will not emit audit logs for any statements they issue.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.log.user_audit = '
+    test_role       ALL
+    another_role    NONE
+    ALL             NONE
+';
+~~~
+
+### User with multiple roles
+
+With the audit settings in this example,
+
+- Users with the username or role `test_role` will not emit audit logs for any statements they issue.
+- Users with the username or role `another_role` will emit audit logs for all statements they issue.
+- All remaining users will not emit audit logs for any statements they issue.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.log.user_audit = '
+    test_role       NONE
+    another_role    ALL
+    ALL             NONE
+';
+~~~
+
+Grant an existing user, `test_user`, both roles `test_role` and `another_role`:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT test_role to test_user;
+GRANT another_role to test_user;
+~~~
+
+If `test_user` executes a statement, an audit log will not be emitted. In this case, CockroachDB would match `test_user` to role `test_role` which is listed first and has the `STATEMENT_FILTER` set to `NONE`. Even though `test_user` is a member of role `another_role` which has `STATEMENT_FILTER` set to `ALL`, CockroachDB does not match `test_user` to that role because it is listed second.
```

Comment:
```suggestion
If `test_user` executes a statement, an audit log will not be emitted. In this case, CockroachDB would match `test_user` to role `test_role`, which is listed first and has the `STATEMENT_FILTER` set to `NONE`. Even though `test_user` is a member of role `another_role` which has `STATEMENT_FILTER` set to `ALL`, CockroachDB does not match `test_user` to that role because it is listed second.
```

Reply:
updated